### PR TITLE
SDK Rework + SAM

### PR DESCRIPTION
**Pull request by PabloSzx**

### CODE REVIEW

File: .changeset/config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@2.1.0/schema.json",
-  "changelog": ["@changesets/changelog-github", { "repo": "soundxyz/sdk" }],
+  "changelog": ["@changesets/changelog-github", { "repo": "soundxyz/sdk-private" }],
```
Comment (PabloSzx) [inline .changeset/config.json:3]:
```suggestion
  "changelog": ["@changesets/changelog-github", { "repo": "soundxyz/sdk" }],
```

File: .changeset/spicy-eggs-wait.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@soundxyz/sdk': minor
+---
+
+SAM
```
Comment (PabloSzx) [inline .changeset/spicy-eggs-wait.md:5]:
```suggestion
SAM + Sound v1.2 support
```

**Review by saihaj — APPROVED** [review]
We also want to write a migration guide if we haven't written one already

**Comment by PabloSzx** [comment]
> We also want to write a migration guide if we haven't written one already

yeah

**Pull request merged**